client/asset/btc/spv_wrapper.go

```diff
@@ -506,7 +506,7 @@ func (w *spvWallet) removePeer(addr string) error {
 
 // syncHeight is the best known sync height among peers.
 func (w *spvWallet) syncHeight() int32 {
-	var maxHeight int32
+	maxHeight := atomic.LoadInt32(&w.syncTarget)
```

Comment:
OK, but it stands out that `syncStatus()` is the only caller of `syncHeight()` and that the only update to this via `atomic.StoreInt32` is in `syncStatus()`.  Sorta suggest that one or the other method should be concerned with the atomic field, but not both.  Otherwise the fix makes sense.